### PR TITLE
Improve the look of webapp

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: src/main/resources/site/static.ts [hunk 1]
```diff
@@ -4,37 +4,30 @@ import type {
...
-import {toStr} from '@enonic/js-utils';
 // @ts-expect-error TS2307: Cannot find module '/lib/enonic/static' or its corresponding type declarations.
 import {buildGetter} from '/lib/enonic/static';
 import { GETTER_ROOT } from '/constants';
+import {DEBUG_MODE} from "/constants";
 
 
 const staticGetter = buildGetter({
-	etag: false, // default is true in production and false in development
+	etag: !DEBUG_MODE, // default is true in production and false in development
```
Comment (ComLock) [inline src/main/resources/site/static.ts:14]: Why?

Comment (alansemenov) [reply src/main/resources/site/static.ts:14]: how can we ensure that people remember to set it to `true` in production?

Comment (ComLock) [reply src/main/resources/site/static.ts:14]: The comment is about what is default for the buildGetter function.

The immuteable getter should never have eTag. Always false.

Comment (ComLock) [reply src/main/resources/site/static.ts:14]: The eTag getter should always have etag: true

**Review by ComLock — APPROVED** [review]

**Pull request merged**